boards/arm/thingy53_nrf5340/thingy53_nrf5340_cpuapp_defconfig

```diff
@@ -24,3 +24,5 @@ CONFIG_UART_CONSOLE=y
 CONFIG_SERIAL=y
 
 CONFIG_PINCTRL=y
+
```

Comment:
Consider adding comments in these configuration files explaining that we enable USB by default in `Kconfig.defaults`, but because of imply in the USB nrfx driver the remote wakeup must be disabled here (looking only at this file the change may look a bit out of context).